example/interleaved_ptr.hpp

```diff
@@ -71,24 +76,36 @@ struct interleaved_ptr : boost::iterator_facade
     interleaved_ptr(reference* pix) : _channels(&((*pix)[0])) {}
     interleaved_ptr& operator=(reference* pix) { _channels=&((*pix)[0]); return *this; }
 
-    /// For some reason operator[] provided by boost::iterator_facade returns a custom class that is convertible to reference
-    /// We require our own reference because it is registered in iterator_traits
-    reference operator[](difference_type d) const { return memunit_advanced_ref(*this,d*sizeof(channel_t));}
+    ///// For some reason operator[] provided by boost::iterator_facade returns a custom class that is convertible to reference
+    ///// We require our own reference because it is registered in iterator_traits
+    //reference operator[](difference_type d) const { return memunit_advanced_ref(*this,d*sizeof(channel_t));}
```

Comment:
I assume this is going to be part of the clean up.

Reply:
Absolutely!